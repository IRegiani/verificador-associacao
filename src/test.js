import React from 'react';
import { shallow } from 'enzyme';
import Routes from './routes';

import LoginPage from './pages/login';

describe('Routes', () => {
  test('renders app correctly', () => {
    const wrapper = shallow(<Routes />);
    expect(wrapper).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });

  test('renders components matching routes', () => {
    const wrapper = shallow(<Routes />);
    const expectedLoginPage = wrapper.childAt(1).props().render();

    expect(expectedLoginPage).toEqual(<LoginPage />);
  });
});
