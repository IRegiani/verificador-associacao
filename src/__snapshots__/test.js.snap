// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes renders app correctly 1`] = `
<Switch>
  <Route
    exact={true}
    path="/"
  >
    <Redirect
      to="/login"
    />
  </Route>
  <Route
    path="/login"
    render={[Function]}
  />
  <Route
    path="*"
  >
    <Redirect
      to="/login"
    />
  </Route>
</Switch>
`;
